### ISSUE TITLE
Button: Extra speech output ‘Button’ is conveyed by the screen reader

### ISSUE THREAD

**Issue by sKudum** [SAP/fundamental-ngx#2964]
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
Screen readers are reading buttons twice on focus of the button.It should read button only once.

Repetitive information shall be avoided & only relevant information should be conveyed by the screen reader.

for example: 
The emphasized button is getting read. as **Emapsized button button** should be read as Empsized button 
The positive button is getting read as **Positive button button**, should be read as Positive button
<img width="548" alt="Screen Shot 2020-08-03 at 9 46 16 AM" src="https://user-images.githubusercontent.com/53487468/89145392-3f19dd80-d56e-11ea-8a50-2ff2a04cfc6f.png">

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use the current version.)
Angular 9

#### If this is a bug, please provide steps for reproducing it.
1). Activate screen reader and focus on it and listen to the screen reader information shared.

It is happening because of button text present on the label.

**Comment by sKudum** [comment]
Hi @InnaAtanasova,@Deno Could you please suggest on this, for consistency we need to have the same labels in core and platform as well.

**Comment by sKudum** [comment]
As per @Deno's suggestion, we won't be removing the 'button' from the text value. as it is just a label and the implantation team won't be implementing it in this way.
